jest.config.ts

```diff
@@ -1,25 +1,18 @@
 import { type JestConfigWithTsJest } from "ts-jest";
 
 const jestConfig: JestConfigWithTsJest = {
-  rootDir: "./",
-  moduleFileExtensions: ["ts", "js"],
+  preset: "ts-jest/presets/default-esm",
+  moduleNameMapper: {
+    "^(\\.{1,2}/.*)\\.js$": "$1",
+  },
```

Comment:
Borrowed this snippet from #170, which matches the recommendations of the [`ts-jest` docs](https://kulshekhar.github.io/ts-jest/docs/guides/esm-support/#use-esm-presets).